### PR TITLE
Input field

### PR DESCRIPTION
**Pull request by Loreen17**
Input fields are done with basic css , I will continue the css later

### CODE REVIEW

**Comment by Loreen17** [comment]
The message unclear can you explain to me again

**Comment by inbalg** [comment]
I'm a little bit confused, because the previous PR (https://github.com/lotus-1/Buddha/pull/3) was merged to master, and yet *this* PR still contains the same commits (all of the navigation bar code).
what happens when you will merge it? I don't completely understand the branch structure (master/sprint-1/front-end) - can you explain it?

File: front-end/src/dropDown/dropDown.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+class DropdownItem extends React.Component {
+  render() {
+    return (
+      <li className="dropdown-option"
+      onClick={this.props.click}>
+        {this.props.label}
+      </li>
+    );
+  }
+  teamLogo;
```
Comment (inbalg) [inline front-end/src/dropDown/dropDown.js]: what's that?

File: front-end/src/dropDown/dropDown.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    this.setState({ toggled: !this.state.toggled });
+  }
+
+  handleClick(value) {
+    this.setState({ selected: value });
+    this.closeDropdown();
+  }
+
+  render() {
+    var className = "dropdown-container";
```
Comment (inbalg) [inline front-end/src/dropDown/dropDown.js]: please use `let` or `const` instead of `var`

File: front-end/src/header/header.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./header.css";
+
+const header = props => {
+  return (
+    <h1>
+          All Content Packages
+          </h1>
```
Comment (inbalg) [inline front-end/src/header/header.js]: indentation please

**Comment by Loreen17** [comment]
Yes this is the branch structure (master/sprint-1/front-end) - I didn't work on the dropdown.
Please review the input files (input.js, input.css)

**Comment by Loreen17** [comment]
Finish working on button review it :hugs:

File: front-end/src/buttons/buttons.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
+.Add {
```
Comment (inbalg) [inline front-end/src/buttons/buttons.css]: the convention for css classes is lower case with `-`. 
(so for example `.create-filter` and not `.CreateFilter`)

File: front-end/src/input/input.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+          <lable>Priority:</lable>
+          <input type="text" onChange={this.handleChange}/>
+          <lable>Package Level 1:</lable>
+          <input className="Level1" type="text" onChange={this.handleChange}/>
+          <br />
+          <lable>Package Level 2:</lable>
+          <input type="text" onChange={this.handleChange}/>
+          <lable>Related WT:</lable>
+          <input type="text" onChange={this.handleChange}/>
+          <lable>Q4`19 Baseline Flag:</lable>
```
Comment (inbalg) [inline front-end/src/input/input.js]: what is "Q4`19 Baseline Flag:"..?

File: front-end/src/input/input.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    return (
+      <div>
+        <form>
+          <lable>Program:</lable>
+          <input type="text" onChange={this.handleChange} />
+          <lable>Priority:</lable>
+          <input type="text" onChange={this.handleChange}/>
+          <lable>Package Level 1:</lable>
+          <input className="Level1" type="text" onChange={this.handleChange}/>
+          <br />
```
Comment (inbalg) [inline front-end/src/input/input.js]: Using `<br/>` is usually considered a bad practice (see this discussion as an example https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br).
See if you can do it with HTML and CSS instead.

**Pull request merged**